.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -90,4 +94,26 @@ stages:
           displayName: Setup Build Variables
         - template: ./templates/.builder-release-template-windows.yaml
           parameters:
-            artifactName: 2022-containerd
\ No newline at end of file
+            artifactName: 2022-containerd
+
+  - stage: build_vhd_2022_containerd_gen2
+    dependsOn: []
+    condition: eq('${{ parameters.build2022containerdgen2 }}', True)
+    jobs:
+    - job: build
+      timeoutInMinutes: 180
+      steps:
+        - bash: |
+            echo '##vso[task.setvariable variable=DRY_RUN]${{parameters.dryrun}}'
+            echo '##vso[task.setvariable variable=HYPERV_GENERATION]V2'
```

Comment:
Why do you remove `echo '##vso[task.setvariable variable=VHD_DEBUG]${{parameters.vhddebug}}'`?

Reply:
`vhddebug` is only used in `.vsts-vhd-builder-windows.yaml` but not this? The repo does not have it in this file. It was not even here in this commit https://github.com/Azure/AgentBaker/commit/b387ed14b8686ee93ea1b19ef23dcb242bf9fdd9#diff-7ff44dd52bb7628aab9652bdb37642f273e4ec43cd48b5e5f4c752bdba31478c.

Reply:
"vhddebug is only used in .vsts-vhd-builder-windows.yaml" yes. I did not see it in .vsts-vhd-builder-windows.yaml. I tagged a wrong file.

Reply:
Added back https://github.com/Azure/AgentBaker/pull/2036/commits/7dfa348264f6453af6a2be809c1b517b66b94de8.